redaxo/src/addons/media_manager/lib/managed_media.php

```diff
@@ -302,8 +302,17 @@ private function prepareHeaders($src)
 
         $header = $this->getHeader();
         if (!isset($header['Content-Type'])) {
-            $finfo = finfo_open(FILEINFO_MIME_TYPE);
-            $content_type = finfo_file($finfo, $this->getSourcePath());
+            $content_type = '';
+            
+            if (!$content_type && function_exists('mime_content_type')) {
+                $content_type = mime_content_type($this->getSourcePath());
+            }            
+                
+            if (!$content_type && function_exists('finfo_open')) {
+               $finfo = finfo_open(FILEINFO_MIME_TYPE);
+               $content_type = finfo_file($finfo, $this->getSourcePath());
```

Comment:
hier fehlt auch `finfo_close($finfo);`, werde ich in separatem PR nachholen da es auch an der anderen stelle fehlt